mypy/test/helpers.py

```diff
@@ -233,6 +233,8 @@ def clean_up(a: List[str]) -> List[str]:
     remove trailing carriage returns.
     """
     res = []
+    pwd = os.getcwd()
```

Comment:
What was going on here?

Reply:
Many tests suddenly showed the full pathname in the traceback instead of simply `File "driver.py", line NNN`. I don't actually know why, but I could imagine that somehow an absolute path was computed by Python where it previously believed the pathname on the command line? It's always about "driver.py" which I presume is the synthetic script created by the test harness.

Reply:
Playing around, this just seems new behavior in Python 3.9, so I think my fix is the best we can do.